### ISSUE TITLE
[Bug] `--watch` is no longer effective

### ISSUE THREAD

**Issue by jasikpark** [storybookjs/test-runner#163]
### Describe the bug

When running `npx test-storybook --watch` the test runner runs in normal mode, rather than watching for changes and exists on completion

### Steps to reproduce the behavior

Bug encountered in a private repo, is there a good starter project I can test deps on?

### Expected behavior

I'd expect `--watch` to watch for file changes and rerun test-storybook accordingly

### Screenshots and/or logs

If applicable, add screenshots and/or logs to help explain your problem.

### Environment

 - OS: macOS
 - Node.js version: v16.16.0
 - NPM version: v8.15.0

```
 "@storybook/test-runner": "^0.6.0-next.1",
 "@storybook/testing-library": "^0.0.13",
 "@storybook/testing-react": "^1.3.0",
 "jest": "^28.1.3",
```

### Additional context

associated discord thread: https://discord.com/channels/486522875931656193/1006156600022667345/1006157098318577714

I think I may have discovered it being due to upgrading to jest 28 in our repo?

**Comment by jasikpark** [comment]
yay https://github.com/storybookjs/test-runner/releases/tag/v0.6.0-next.3

**Comment by yannbf** [comment]
Fix was released in [v0.6.0](https://github.com/storybookjs/test-runner/releases/tag/v0.6.0)! Thanks once again @jasikpark
